github-pullrequest-plugin/src/main/java/com/github/kostyasha/github/integration/branch/GitHubBranchRepository.java

```diff
@@ -82,6 +82,12 @@ public String getUrlName() {
     }
 
 
+    public synchronized void removeBranch(String branchName) {
+        if (nonNull(branches)) {
```

Comment:
you can just `getBranches().remove(branchName)` and i guess `@Nonnull String branchName` argument